src/server.cpp

```diff
@@ -333,6 +330,22 @@ Server::~Server()
 		}
 		m_env->kickAllPlayers(SERVER_ACCESSDENIED_SHUTDOWN,
 			kick_msg, reconnect);
+	}
+
+	// Stop all emerge threads before deleting players that may have requested
+	// blocks to be emerged.
```

Comment:
No longer valid.

Reply:
Then it wasn't valid before either, since I moved it up. I don't know when it was last valid (will fix though)